tests/contrib/openai/test_openai.py

```diff
@@ -0,0 +1,171 @@
+from typing import List
+from typing import Optional
+
+import openai
+import pytest
+
+from ddtrace import Pin
+from ddtrace import Span
+from ddtrace import patch
+from ddtrace.contrib.openai.patch import unpatch
+from ddtrace.filters import TraceFilter
+from tests.contrib.openai.openai_vcr import openai_vcr
+
+
+class FilterOrg(TraceFilter):
+    def process_trace(self, trace):
+        # type: (List[Span]) -> Optional[List[Span]]
+        for span in trace:
+            if span.get_tag("organization"):
+                span.set_tag_str("organization", "not-a-real-org")
+            else:
+                raise ValueError("span must have tag organization")
+        return trace
+
+
+@pytest.fixture(autouse=True)
+def patch_openai():
+    # FIXME: aiohttp spans are not being generated in these tests, it looks like they should be
```

Comment:
this comment can be removed once we add a `test_integration_async` test case below that captures the aiohttp spans

@lievan to own.